types/ApiResponses.ts

```diff
@@ -303,6 +304,7 @@ export interface PrivateProfile extends PublicProfile {
   accessType: string,
   state: string,
   credits: number,
+  resumes: ResumeModel[],
```

Comment:
nit: this should also be `PublicResume[]`, since in almost every case we don't want to return a model object to the client since it might display fields that we didn't intend on showing (e.g. `id`, `UserModel`, etc). Even though this is a "private profile", we still return the "public" type because the term "public" in our types refers to anything the client interacts with